packages/react-core/src/components/Dropdown/DropdownItem.tsx

```diff
@@ -89,6 +92,7 @@ export const DropdownItem: React.FunctionComponent<DropdownItemProps> = ({
         autoFocus={autoFocus}
         styleChildren={styleChildren}
         description={description}
+        {...getOUIAProps(DropdownItem.displayName, ouiaId, ouiaSafe)}
```

Comment:
for better default ids you can follow this comment https://github.com/patternfly/patternfly-react/pull/4769#discussion_r488157601

Reply:
I'm afraid my poor knowledge of TS is not enough to apply the same trick as in  https://github.com/patternfly/patternfly-react/pull/4769#discussion_r488157601. Where should I put `const ouiaProps = useOUIAProps(DropdownItem.displayName, ouiaId, ouiaSafe);`?

Reply:
You should change the signature from:
```tsx
export const DropdownItem = ({
  ...
}) => (
  <element />
)
```
to
```tsx
export const DropdownItem = ({
  ...
}) => {
  const ouiaProps = useOUIAProps(NavItem.displayName, ouiaId, ouiaSafe);
  return (
    <element {...ouiaProps} />
  )
}
```

Reply:
@redallen thank you very much!